pkg/implementer.go

```diff
@@ -0,0 +1,329 @@
+package impl
+
+import (
+	"bytes"
+	"fmt"
+	"go/ast"
+	"go/build"
+	"go/format"
+	"go/parser"
+	"go/token"
+	"io"
+	"io/ioutil"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"golang.org/x/tools/go/buildutil"
+)
+
+// An Implementer can, for a certain directory, create and/or update
+// implementation with Go source code for a particular interface
+type Implementer struct {
+	Recv, IFace string
+
+	//Dir is the working directory to search for the receiver
```

Comment:
Minor, there should be a space after the two slashes.

Reference: https://dmitri.shuralyov.com/idiomatic-go#comments-for-humans-always-have-a-single-space-after-the-slashes.